<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataInputText</class>
 <widget class="QFrame" name="DataInputText">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(81, 81, 81);</string>
  </property>
  <widget class="QLabel" name="labelMode">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>文本读取模式：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSave">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>570</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(100, 100, 100);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>保存数据</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSaveInfo">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>580</y>
     <width>531</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>保存数据成功！（20:00:00）</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonExit">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>635</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(100, 100, 100);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>返回首页</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTitle">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>471</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>数据写入页面</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxMode">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>411</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(100, 100, 100);
color: rgb(255, 255, 255);</string>
   </property>
   <item>
    <property name="text">
     <string> 奇数行解释为中文 偶数行解释为英文（先中后英）</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string> 奇数行解释为英语 偶数行解释为中文（先英后中）</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="labelWindow">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>文本输入窗口：</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEditWindow">
   <property name="geometry">
    <rect>
     <x>52</x>
     <y>260</y>
     <width>681</width>
     <height>261</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="plainText">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
